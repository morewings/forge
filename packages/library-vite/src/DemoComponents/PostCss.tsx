import React, { FC } from 'react';
import classes from './postcss.module.css';

export const PostCss: FC = () => {
    return (
        <div className={classes.demo}>
            red <div className={classes.inside}>green</div>
        </div>
    );
};
