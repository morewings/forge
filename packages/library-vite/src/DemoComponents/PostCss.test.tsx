import { render } from '@testing-library/react';
import { PostCss } from './PostCss';

describe('PostCss', () => {
    it('renders', () => {
        const { asFragment } = render(<PostCss />);
        expect(asFragment()).toMatchSnapshot();
    });
});
