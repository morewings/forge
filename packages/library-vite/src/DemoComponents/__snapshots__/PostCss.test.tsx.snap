// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostCss renders 1`] = `
<DocumentFragment>
  <div>
    red 
    <div>
      green
    </div>
  </div>
</DocumentFragment>
`;
